src/main/webapp/style.css

```diff
@@ -1,7 +1,49 @@
+body, html {
```

Comment:
I think you can delete this file now, since you moved the shared CSS to css/style.css instead?